boards/px4/fmu-v3/init/rc.board_sensors

```diff
@@ -15,6 +15,12 @@ hmc5883 -C -T -I -R 4 start
 # Internal SPI bus ICM-20608-G
 mpu6000 -T 20608 start
 
+# External SPI
+ms5611 -S start
+
+# Internal SPI (auto detect ms5611 or ms5607)
```

Comment:
@dagar I was under the impression the detection was need on for Intel Aero. Also if it is ONLY needed on certain builds then let's ifdef the code and save the code space. Drop the parameter -T 0 and  ADD #BOARD_CAN_HAVE_MS5607 to the board_config.h then if def the code and do the test always.